.github/workflows/performancetest.yml

```diff
@@ -0,0 +1,42 @@
+name: Performance-Test
+
+on:  
+  push:
+    branches:
+      - development
+
+jobs:
+  slsart-perf:
+    name: slsart Performance Test
+    runs-on: ubuntu-latest
+    env:
+      NPM_CONFIG_PREFIX: "~/.npm-global"
+      STAGE_NAME: merge-perf-test
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
I chose v14 as well, I believe we can have multiple versions here just in case there are variances but im not sure it matters

Reply:
Yes, we can also use a matrix if we want to launch a separate job for each version.